Binary files change-01-fixed_commandline_and_comments/.git/index and change-02-update_version_and_credits/.git/index differ
diff -rudN change-01-fixed_commandline_and_comments/src/commandline.cc change-02-update_version_and_credits/src/commandline.cc
--- change-01-fixed_commandline_and_comments/src/commandline.cc	2013-08-12 01:14:34.849552333 +0200
+++ change-02-update_version_and_credits/src/commandline.cc	2013-08-12 01:15:47.909551080 +0200
@@ -17,7 +17,8 @@
 #include "src/film.h"
 #include "src/xml.h"
 
-#define APP_VERSION "SVN-r$Revision: 194 $-$Author: peterb $"
+#define APP_VERSION "2.0.0-git"     // This is also being declared in "main.cc"
+
 class xml;
 class film;
 
@@ -44,7 +45,7 @@
 void
 show_help (char **argv)
 {
-  printf ("\nShotdetect version \"%s\" ($Date: 2010-10-01 01:35:11 +0200 (Fri, 01 Oct 2010) $), Copyright (c) 2007-2010 Johan Mathe\n\n"
+  printf ("\nShotdetect version \"%s\", Copyright (c) 2007-2013 Johan Mathe\n\n"
           "Usage: %s \n"
           "-h           : show this help\n"
           "-s threshold : threshold\n"
diff -rudN change-01-fixed_commandline_and_comments/src/main.cc change-02-update_version_and_credits/src/main.cc
--- change-01-fixed_commandline_and_comments/src/main.cc	2013-08-12 00:45:32.593582189 +0200
+++ change-02-update_version_and_credits/src/main.cc	2013-08-12 00:54:24.293573077 +0200
@@ -22,7 +22,7 @@
 #include "src/ui/dialog_shotdetect.h"
 #include "src/xml.h"
 
-#define APP_VERSION "SVN-r$Revision: 194 $-$Author: peterb $"
+#define APP_VERSION "2.0.0-git"     // This is also being declared in "commandline.cc"
 
 class xml;
 class film;
@@ -41,7 +41,7 @@
 }
 
 void show_help (char **argv) {
-  printf ("\nShotdetect version \"%s\" ($Date: 2010-10-01 01:35:11 +0200 (Fri, 01 Oct 2010) $), Copyright (c) 2007-2010 Johan Mathe\n\n"
+  printf ("\nShotdetect version \"%s\", Copyright (c) 2007-2013 Johan Mathe\n\n"
           "Usage: %s \n"
           "-h           : show this help\n"
           "-n           : commandline mode (disable GUI)\n"
@@ -173,7 +173,6 @@
     default:
       break;
     }
-
   }
 
   if (!gui) {
